src/config/content/ca.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * The default limit of 20 words is too strict for Catalan. For more detailed explanation, see:
```

Comment:
Comments that focus on the _why_ rather than the _what_ are often better.
You can tell by inspecting the code that the default word count is 20 and the new count is 25, but you cannot tell why this is the case 😉.
Perhaps this is a better description:
```suggestion
 * Sentences in Catalan have a similar length distribution as in Spanish, so use the same 25 word limit. For a more detailed explanation, see:
```